### ISSUE TITLE
JQuery Core is organized in the wrong order

### ISSUE THREAD

**Issue by namel** [jquery/learn.jquery.com#393]
Hi

I've been learning jquery through the "JQuery Core" pages.  It's great,
but it teaches things in the wrong order.

Pages "selecting elements", and "working with selections"
should be the first pages.
Then, pages like "$ vs. $()" and `$(document).ready()` which are the first two pages (!), should appear much later - since the reader has no idea what those things mean yet...  at least I didn't.

Basically - the first 4 pages should be moved to the end in the sequence.

Also, the first page should probably be a "getting started page" which
shows the basic abilities.  Maybe a repeat of "How JQuery Works" page from the intro.

**Comment by TaMe3971** [comment]
Hey is anyone working on this?

**Comment by ajpiano** [comment]
Not at the moment!

**Comment by TaMe3971** [comment]
Cool, I will take a look.
